### PR TITLE
Bugfix/bsc 964618

### PR DESCRIPTION
**Pull request by taroth21**
@toabctl : Please review my additions to xml/book_cloud_suppl.xml until end of this week, if possible. If you have feedback, please add comments to the PR in GH. Thanks!

I'll merge the PR into maintenance/openstack_cloud_6 on Friday (Dec 23).

### CODE REVIEW

File: xml/book_cloud_suppl.xml [hunk 1]
```diff
@@ -881,6 +881,82 @@
...
+    boot the Btrfs image with a custom kernel to start the instances. Prepare
+    the Btrfs image as described in
+    <xref
+     linkend="pro.suppl.img.brtfs.custom.kernel"/>.
+   </para>
+
+   <procedure xml:id="pro.suppl.img.brtfs.custom.kernel">
+    <title>Booting Btrfs Images with a Custom Kernel</title>
+    <para>
+     Execute the following on your &contrnode;:
```
Comment (toabctl) [inline xml/book_cloud_suppl.xml]: The commands can be executed anywhere. You just need:
- credentials for a OpenStack project
- glanceclient or openstackclient to upload the images
- the ```grub.xen``` file (which you find on SLES systems)

File: xml/book_cloud_suppl.xml [hunk 2]
```diff
@@ -881,6 +881,82 @@
...
+     <para>
+      Create a &o_img; image with the kernel from this package. For example:
+     </para>
+<screen>glance image-create --name img-grub-xen-x64 \
+ --file /usr/lib/grub2/x86_64-xen/grub.xen --is-public True \
+ --container-format bare --disk-format raw</screen>
+    </step>
+    <step>
+     <para>
+      Create a second image with the Btrfs file system. For example:
```
Comment (toabctl) [inline xml/book_cloud_suppl.xml]: To be more precise, ```Create a second image (which uses Btrfs as root file system)``` or something like this.

File: xml/book_cloud_suppl.xml (hunk 1 above)
Comment (taroth21) [inline xml/book_cloud_suppl.xml]: Thanks! Added this to the requirements for the procedure (apart from grub.xen, which I decided to mention in step 1 where the ```grub2-xen``` package is installed).

File: xml/book_cloud_suppl.xml (hunk 2 above)
Comment (taroth21) [inline xml/book_cloud_suppl.xml]: Done, thanks!

**Comment by taroth21** [comment]
Merged into maintenance/openstack_cloud_6 with https://github.com/SUSE-Cloud/doc-cloud/commit/4a029f5.

**Pull request closed without merging**